src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -41,41 +41,33 @@ public static class BitOperations
             19, 27, 23, 06, 26, 05, 04, 31
         };
 
-        private static readonly uint[] s_crcTable = new uint[]
+        private static readonly uint[] s_crcTable = GenerateReflectedTable(0x82F63B78u);
```

Comment:
i meant reuse the code instead of duplicating it https://github.com/kasperk81/runtime/commit/d74d96a9002e6691ef8d501c80e6cd7669db5097